lib/services/censored_log.dart

```diff
@@ -27,9 +27,13 @@ extension CensoredMessage on LogRecord {
 
       if (user != null) {
         workingLogString = workingLogString.replaceAll(
-            CaseInsensitivePattern(user.baseUrl), "BASEURL");
+            CaseInsensitivePattern(user.baseURL), "ACTIVEURL").replaceAll(
```

Comment:
I know you didn't change this, but what do you think about always resolving if it's the local or remote URL? That might help more with debugging than seeing `ACTIVEURL` and having to figure out which URL was switched to last.  
Alternatively we could put put an `active` qualifier in parentheses to confirm that this was used for the current request, and isn't some error from an old request that was made before the switch.